frontend/scripts/react-components/dashboard-element/dashboard-widget/table-modal/table-modal.selectors.js

```diff
@@ -0,0 +1,149 @@
+import { createSelector } from 'reselect';
+import isEmpty from 'lodash/isEmpty';
+import { format } from 'd3-format';
+import { CHART_TYPES } from 'constants';
+
+const getMeta = (state, { meta }) => meta || null;
+const getData = (state, { data }) => data || null;
+const getChartType = (state, { chartType }) => chartType || null;
+const returnNameArrayIfNotEmpty = value =>
+  isEmpty(value) ? null : Object.values(value).map(v => v.name);
+const getCountriesPanel = state =>
+  returnNameArrayIfNotEmpty(
+    state.dashboardElement && state.dashboardElement.countriesPanel.activeItems
+  );
+const getCommoditiesPanel = state =>
```

Comment:
Should this and the countries be an array?

Reply:
They are an array 🤷‍♂️

Reply:
exactly haha. Sorry I meant to say: Shouldn't this be an item or null.

Reply:
ActiveItems is an array in all cases. As the purpose is to have several countries and commodities I think it makes sense to keep them all as arrays. Nevertheless we will always have a country and a commodity

Reply:
Not sure I love being this smart into the future, but I can understand your point. Oh well, let's leave it as is.